docs/source/gateway/index.rst

```diff
@@ -1,6 +1,1049 @@
+With the rapid development of LLMs, there is no guarantee that this list will be up to date at all times. However, the associations listed
+below can be used as a helpful guide when configuring a given route for any newly released model types as they become available with a given provider.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Route Type
+     - Provider
+     - Model Examples
+     - Supported
+   * - llm/v1/completions
+     - OpenAI
+     - gpt-3.5-turbo, gpt-4
+     - ✓
+   * - llm/v1/completions
+     - Anthropic
+     - claude-1, claude-1.3-100k
+     - ✓
+   * - llm/v1/completions
+     - Cohere
+     - command, command-light-nightly
+     - ✓
+   * - llm/v1/completions
+     - Azure OpenAI
+     - text-davinci-003, gpt-35-turbo
+     - ✓
+   * - llm/v1/chat
+     - OpenAI
+     - gpt-3.5-turbo, gpt-4
+     - ✓
+   * - llm/v1/chat
+     - Anthropic
+     -
+     - ✗
+   * - llm/v1/chat
+     - Cohere
+     -
+     - ✗
+   * - llm/v1/chat
+     - Azure OpenAI
+     - gpt-35-turbo, gpt-4
+     - ✓
+   * - llm/v1/embeddings
+     - OpenAI
+     - text-embedding-ada-002
+     - ✓
+   * - llm/v1/embeddings
+     - Anthropic
+     -
+     - ✗
+   * - llm/v1/embeddings
+     - Cohere
+     - embed-english-v2.0, embed-multilingual-v2.0
+     - ✓
+   * - llm/v1/embeddings
+     - Azure OpenAI
+     - text-embedding-ada-002
+     - ✓
+
+Within each model block in the configuration file, the provider field is used to specify the name
+of the provider for that model. This is a string value that needs to correspond to a provider the MLflow AI Gateway supports.
+
+Here's an example of a provider configuration within a route:
+
+.. code-block:: yaml
+
+    routes:
+        - name: chat
+          type: llm/v1/chat
+          model:
+            provider: openai
+            name: gpt-4
+            config:
+              openai_api_base: https://api.openai.com/v1
+              openai_api_key: $OPENAI_API_KEY
+
+In the above configuration, ``openai`` is the `provider` for the model.
+
+As of now, the MLflow AI Gateway supports the following providers:
+
+* **openai**: This is used for models offered by `OpenAI <https://platform.openai.com/>`_.
+* **anthropic**: This is used for models offered by `Anthropic <https://docs.anthropic.com/claude/docs>`_.
+* **cohere**: This is used for models offered by `Cohere <https://docs.cohere.com/docs>`_.
+* **azure** / **azuread**: This is used for model services offered by `Azure <https://learn.microsoft.com/en-gb/azure/cognitive-services/openai/>`_.
+
+More providers are being added continually. Check the latest version of the MLflow AI Gateway Docs for the
+most up-to-date list of supported providers.
+
+Remember, the provider you specify must be one that the MLflow AI Gateway supports. If the provider
+is not supported, the Gateway will return an error when trying to route requests to that provider.
```

Comment:
IMO these should go to the top of the `Concepts` section

Reply:
agreed :)